api/src/db/models/entity_models.py

```diff
@@ -0,0 +1,69 @@
+import uuid
+from datetime import date
+
+from sqlalchemy import ForeignKey, Integer, Text
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from src.adapters.db.type_decorators.postgres_type_decorators import LookupColumn
+from src.db.models.base import ApiSchemaTable, TimestampMixin
+from src.db.models.lookup_models import LkSamGovImportType
+
+
+class SamGovEntity(ApiSchemaTable, TimestampMixin):
+    __tablename__ = "sam_gov_entity"
+
+    sam_gov_entity_id: Mapped[uuid.UUID] = mapped_column(UUID, primary_key=True, default=uuid.uuid4)
+    uei: Mapped[str] = mapped_column(Text, nullable=False, index=True)
```

Comment:
The type/nullability can be excluded as they are based on the python typing.
```suggestion
    uei: Mapped[str] = mapped_column(index=True)
```